### ISSUE TITLE
[BUG] Closing setting pane menu button is wrong

### ISSUE THREAD

**Issue by livcheerful** [livcheerful/anywhereadventures#319]
**Describe the bug**
The text that is read out is "Open Menu" when the settings menu is open. It should be saying "Close Menu"

TIMESTAMP - 1:02
